manage.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import logging
+import click
+from app import create_app, models, views, admin
+from app.db import db
+
+app = create_app()
+
+if app.config.get("LOGGER_ENABLED"):
+    logging.basicConfig(
+        level=getattr(logging, app.config.get("LOGGER_LEVEL", "DEBUG")),
+        format=app.config.get(
+            "LOGGER_FORMAT",
+            '%(asctime)s %(name)-12s %(levelname)-8s %(message)s'),
+        datefmt=app.config.get("LOGGER_DATE_FORMAT", '%d.%m %H:%M:%S')
+    )
+
+
+@click.group()
+def core_cmd():
+    """ Core commands wrapper """
+    pass
+
+
+@core_cmd.command()
+@click.option('--ipython/--no-ipython', default=True)
+def shell(ipython):
+    """Runs a Python shell with QPD context"""
+    import code
```

Comment:
we can, but maybe it is not a good idea, because **code** for example is only needed for **shell** command and if we are running other commands we will be importing useless things...